app/helpers/orders_helper.rb

```diff
@@ -0,0 +1,8 @@
+module OrdersHelper
```

Comment:
It is excellent that you have moved this logic out of view, check out presenters and/or decorators :)